crates/runtime/src/continuation.rs

```diff
@@ -329,6 +329,7 @@ pub fn resume(
             // calling trampoline to execute it.
 
             unsafe { (*contobj).state = State::Returned };
+            drop_cont_obj(contobj);
```

Comment:
This doesn't quite work: If we continuation has return values, we read them from the `ContinuationObject` (more concretely: the `args` buffer therein) after returning. See the call to `environ.typed_continuations_load_return_values` in the `Operator::Resume` case in `code_translator.rs`.
This may have worked in some examples without segfaults because `typed_continuations_load_return_values` is a noop if there are no expected return values according to the continuation's type.

What we could always safely do here is deallocating the `Fiber` object.